ground/gcs/src/plugins/config/configautotunewidget.h

```diff
@@ -122,17 +129,20 @@ class ConfigAutotuneWidget : public ConfigTaskWidget
     ConfigGadgetWidget *parentConfigWidget;
     static const QString databaseUrl;
 
-    QString systemIdentValid(SystemIdent::DataFields &data, bool *okToContinue);
+    QString tuneValid(AutotunedValues &data, bool *okToContinue);
```

Comment:
Method can be const? It's a bit of a weird method tbh. It would be much better to return a bool and pass a ref to an error string;